.cqfd/docker/Dockerfile

```diff
@@ -14,3 +14,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	python3-pil \
 	python3-pycparser \
 	python3-serial
+
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	wget \
+	unzip
+
+RUN ln -nsf /usr/bin/python3 /usr/local/bin/python
```

Comment:
Hmm, can you point out what exactly was using the wrong python version?

Reply:
stm32/blackmagic: GENHDR invokes as `/usr/bin/env python`. It appears to be compatible with python3 since the build passes after kludging the symlink.

```sh
[...]
make -C blackmagic PROBE_HOST=nsec19                                                                             │  set mem inaccessible-by-default off                                                                           
make[2]: Entering directory '/home/bengardiner/src/nsec-badge-2019_workshop/stm32/blackmagic'                    │  set debug arm                                                                                                 
  GENHDR  include/libopencm3/efm32/g/irq.json                                                                    │  echo attaching\n                                                                                              
/usr/bin/env: 'python': No such file or directory                                                                │  attach 1                                                                                                      
Makefile:54: recipe for target 'include/libopencm3/efm32/g/irq.json.genhdr' failed                               │end                                                                                                             
make[3]: *** [include/libopencm3/efm32/g/irq.json.genhdr] Error 127                                              │                                                                                                                
Makefile:7: recipe for target 'all' failed                                                                       │if $LOAD_SOFTDEVICE                                                                                             
make[2]: *** [all] Error 2                                                                                       │  load nordicsdk/softdevice/s132/hex/s132_nrf52_5.0.0_softdevice.hex                                            
make[2]: Leaving directory '/home/bengardiner/src/nsec-badge-2019_workshop/stm32/blackmagic'                     │end                                                                                                             
Makefile:15: recipe for target 'blackmagic/src/blackmagic' failed                                                │                                                                                                                
make[1]: *** [blackmagic/src/blackmagic] Error 2                                                                 │load builds/nsec19_nrf52_ctf.out                                                                                
make[1]: Leaving directory '/home/bengardiner/src/nsec-badge-2019_workshop/stm32'                                │file builds/nsec19_nrf52_ctf.out                                                                                
Makefile:9: recipe for target 'builds/nsec19_stm32_debugger.elf' failed                                          │                                                                                                                
make: *** [builds/nsec19_stm32_debugger.elf] Error 2
[...]
```

Reply:
Ah, I would have suggested replacing the shebang line to put `python3` instead of `python`, but this is in a repo we don't own (libopencm3).  And it's not really worth making a fork just for that.

I'm fine with your workaround.  Another one would be to install python (2) in the docker image.